fastify.js

```diff
@@ -187,6 +189,9 @@ function fastify (options) {
     [pluginUtils.registeredPlugins]: [],
     [kPluginNameChain]: [],
     [kAvvioBoot]: null,
+    // routing method
+    routing: httpHandler,
+    defaultRoute: defaultRoute,
```

Comment:
why is this exposed?

Reply:
More details are available in this PR [fastify/fastify-websocket/#95](https://github.com/fastify/fastify-websocket/pull/95), but to sum up:
- Routing utility is exposed so that `fastify-websocket` can dispatch incoming websocket connection requests through the fastify request lifecycle.
- To support global handler, non-matched requests are handled by the `defaultRoute` which needs to be updated by `fastify-websocket` to handle websocket connections as well.

Reply:
A more performant approach is to update `.defaultRoute` on the router itself (find-my-way) via a setter and getter combination.